src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskReplicationMIControllerTest.java

```diff
@@ -402,16 +542,117 @@ public void user_should_claim_task_and_claim_action_recorded_in_reportable_task(
+            .body("reportable_task_list.size()", equalTo(1))
+            .body("reportable_task_list.get(0).state", equalTo("COMPLETED"))
+            .body("reportable_task_list.get(0).assignee", notNullValue())
+            .body("reportable_task_list.get(0).updated_by", notNullValue())
+            .body("reportable_task_list.get(0).updated", notNullValue())
+            .body("reportable_task_list.get(0).update_action", equalTo("Complete"))
+            .body("reportable_task_list.get(0).completed_date", notNullValue())
+            .body("reportable_task_list.get(0).completed_date_time", notNullValue())
+            .body("reportable_task_list.get(0).created_date", notNullValue())
+            .body("reportable_task_list.get(0).due_date", notNullValue())
+            .body("reportable_task_list.get(0).last_updated_date", notNullValue())
+            .body("reportable_task_list.get(0).final_state_label", equalTo("COMPLETED"))
+            .body("reportable_task_list.get(0).first_assigned_date", notNullValue())
+            .body("reportable_task_list.get(0).first_assigned_date_time", notNullValue())
+            .body("reportable_task_list.get(0).wait_time_days", notNullValue())
+            .body("reportable_task_list.get(0).wait_time", notNullValue())
+            .body("reportable_task_list.get(0).handling_time_days", notNullValue())
+            .body("reportable_task_list.get(0).handling_time", notNullValue())
+            .body("reportable_task_list.get(0).processing_time_days", notNullValue())
+            .body("reportable_task_list.get(0).processing_time", notNullValue())
+            .body("reportable_task_list.get(0).is_within_sla", equalTo("Yes"))
+            .body("reportable_task_list.get(0).number_of_reassignments", equalTo(0))
+            .body("reportable_task_list.get(0).due_date_to_completed_diff_days", equalTo(-10))
+            .body("reportable_task_list.get(0).due_date_to_completed_diff_time", notNullValue());
+
+        resultAssignments = restApiActions.get(
+            ENDPOINT_BEING_TESTED_ASSIGNMENTS,
+            taskId,
+            caseworkerCredentials.getHeaders()
+        );
+        resultAssignments.then().assertThat()
+            .statusCode(HttpStatus.OK.value())
+            .body("task_assignments_list.size()", equalTo(1))
+            .body("task_assignments_list.get(0).service", equalTo("WA"))
+            .body("task_assignments_list.get(0).location", equalTo("765324"))
+            .body("task_assignments_list.get(0).assignment_start", notNullValue())
+            .body("task_assignments_list.get(0).assignment_end", notNullValue())
+            .body("task_assignments_list.get(0).assignee", notNullValue())
+            .body("task_assignments_list.get(0).role_category", equalTo("LEGAL_OPERATIONS"))
+            .body("task_assignments_list.get(0).task_name", equalTo("Process Application"))
+            .body("task_assignments_list.get(0).assignment_end_reason", equalTo("COMPLETED"));
```

Comment:
Similar comments to above in that we should try to check dates along with expected assignee where possible